packs/data/spell-effects.db/spell-effect-evolution-surge.json

```diff
@@ -0,0 +1,140 @@
+                        "value": "large"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.SpellEffectEvolutionSurge.Climb",
+                        "value": "climb"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.SpellEffectEvolutionSurge.Huge",
+                        "value": "huge"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.SpellEffectEvolutionSurge.Fly",
+                        "value": "fly"
+                    }
+                ],
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.SpellEffectEvolutionSurge.Prompt",
+                "rollOption": "evolution-surge"
+            },
+            {
+                "key": "Sense",
+                "predicate": "evolution-surge:sight",
+                "selector": "lowLightVision"
+            },
+            {
+                "key": "Sense",
+                "predicate": "evolution-surge:sight",
+                "selector": "darkvision"
+            },
+            {
+                "acuity": "imprecise",
+                "key": "Sense",
+                "predicate": "evolution-surge:scent",
+                "range": 30,
+                "selector": "scent"
+            },
+            {
+                "add": [
+                    "amphibious"
+                ],
+                "key": "ActorTraits",
+                "predicate": "evolution-surge:amphibious"
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": "evolution-surge:speed",
+                "selector": "all-speeds",
+                "type": "status",
+                "value": 20
+            },
+            {
+                "key": "CreatureSize",
+                "predicate": "evolution-surge:large",
+                "reach": {
+                    "upgrade": 10
+                },
+                "value": "large"
+            },
+            {
+                "key": "BaseSpeed",
+                "predicate": "evolution-surge:climb",
+                "selector": "climb",
+                "value": "@actor.attributes.speed.total"
+            },
+            {
+                "key": "CreatureSize",
+                "predicate": "evolution-surge:huge",
+                "reach": {
+                    "upgrade": 15
+                },
+                "value": "huge"
+            },
+            {
+                "key": "BaseSpeed",
+                "predicate": "evolution-surge:fly",
+                "selector": "fly",
+                "value": "@actor.attributes.speed.total"
+            },
+            {
+                "key": "BaseSpeed",
+                "predicate": "evolution-surge:amphibious",
+                "selector": "swim",
+                "value": "min(@actor.attributes.speed.total, 30)"
+            }
```

Comment:
```suggestion
                "predicate": ["evolution-surge:sight"],
                "selector": "lowLightVision"
            },
            {
                "key": "Sense",
                "predicate": ["evolution-surge:sight"],
                "selector": "darkvision"
            },
            {
                "acuity": "imprecise",
                "key": "Sense",
                "predicate": ["evolution-surge:scent"],
                "range": 30,
                "selector": "scent"
            },
            {
                "add": [
                    "amphibious"
                ],
                "key": "ActorTraits",
                "predicate": ["evolution-surge:amphibious]"
            },
            {
                "key": "FlatModifier",
                "predicate": ["evolution-surge:speed"],
                "selector": "all-speeds",
                "type": "status",
                "value": 20
            },
            {
                "key": "CreatureSize",
                "predicate": ["evolution-surge:large"],
                "reach": {
                    "upgrade": 10
                },
                "value": "large"
            },
            {
                "key": "BaseSpeed",
                "predicate": ["evolution-surge:climb"],
                "selector": "climb",
                "value": "@actor.attributes.speed.total"
            },
            {
                "key": "CreatureSize",
                "predicate": ["evolution-surge:huge"],
                "reach": {
                    "upgrade": 15
                },
                "value": "huge"
            },
            {
                "key": "BaseSpeed",
                "predicate": ["evolution-surge:fly"],
                "selector": "fly",
                "value": "@actor.attributes.speed.total"
            },
            {
                "key": "BaseSpeed",
                "predicate": ["evolution-surge:amphibious"],
                "selector": "swim",
                "value": "min(@actor.attributes.speed.total, 30)"
            }
```
Predicates are arrays.

Reply:
I'll make the change, but I think it's worth noting this works as-is, and the console does not give any warnings or errors. If predicates not being arrays is incorrect, there should probably be some sort of user feedback right?

Reply:
There should be, but I think right now it would just fail to see the predicate and always be true.